MockObjects/Objects/MockObjectsFiles/MockRepositories.cpp

```diff
@@ -0,0 +1,312 @@
+#include "MockRepositories.hpp"
+
+#include "DataAccess.Postgre/UsersAmountFinder.hpp"
+#include "Network/Connection.hpp"
+#include "Models/Primitives.hpp"
+
+#include "Cryptography.hpp"
+#include "FileLogger.hpp"
+
+namespace MockRepositories
+{
+std::vector<Models::ChannelInfo> testChannelsRepository::getAllChannelsList()
+{
+    Models::ChannelInfo channelInfo;
+
+    channelInfo._channelID   = 15;
```

Comment:
This data has to be stored inside of a data files nearby or entered during test execution.

Reply:
Here and below.

Reply:
I understand, I was following the task set for me by Daniel